src/AmbientSounds.Uwp/Controls/TitleBarLogo.xaml.cs

```diff
@@ -12,9 +13,22 @@ public sealed partial class TitleBarLogo : UserControl
             typeof(TitleBarLogo),
             new PropertyMetadata(Strings.Resources.AppDisplayName));
 
+    public static readonly DependencyProperty IsWindowFocusedProperty =
+        DependencyProperty.Register("IsWindowFocused", typeof(bool), typeof(TitleBarLogo), new PropertyMetadata(false));
+
     public TitleBarLogo()
     {
         this.InitializeComponent();
+        Window.Current.Activated += (_, e) =>
+        {
+            IsWindowFocused = e.WindowActivationState != CoreWindowActivationState.Deactivated;
+        };
```

Comment:
This leads to a memory leak. Repro:
1. Launch ambie
2. navigate to screenshots page
3. press back button to return to home page
4. Observe that this event handler gets called twice (because it was registered twice)

To fix this, please do the following
1. Create event handlers for this control's `Loaded` and `Unloaded` events
2. Create a dedicated method that handles the window activated event
3. Using the dedicated method, add a `Windows.Current.Activated` registration to the `OnLoaded` event handler.
4. Add a unregister line in the `OnUnloaded` event handler

Once you've made the changes, run the repro steps from earlier. There shouldn't be a duplicate registration anymore.

Reply:
Done.